src/components/group.ts

```diff
@@ -3,6 +3,7 @@
 module Plottable {
 export module Components {
   export class Group extends ComponentContainer {
+    private _componentList: Component[] = [];
```

Comment:
`_componentList` is the internal list; `_components()` is the method called by the parent class to get the full list of `Component`s. `components()` is the public-facing version.